README.md

```diff
@@ -99,20 +99,23 @@ The commands above can also be used to update Trixi. A brief list of notable
 changes to Trixi is available in [`NEWS.md`](NEWS.md).
 
 ### For developers
-If you plan on editing Trixi itself, you can download Trixi locally and run it from
-within the cloned directory:
+If you plan on editing Trixi itself, you can download Trixi locally and use the
+code from the cloned directory:
 ```bash
 git clone git@github.com:trixi-framework/Trixi.jl.git
 cd Trixi.jl
-julia --project=@. -e 'import Pkg; Pkg.instantiate()' # Install Trixi's dependencies
-julia -e 'import Pkg; Pkg.add(["Trixi2Vtk", "Plots"])' # Install postprocessing tools
-julia -e 'import Pkg; Pkg.add("OrdinaryDiffEq")' # Install time integration schemes
+mkdir run 
+cd run
+julia --project=. -e 'using Pkg; Pkg.develop(PackageSpec(path=".."))' # Install local Trixi clone
+julia -e 'using Pkg; Pkg.add(["Trixi2Vtk", "Plots"])' # Install postprocessing tools
+julia -e 'using Pkg; Pkg.add("OrdinaryDiffEq")' # Install time integration schemes
```

Comment:
It would save time if you merged these two lines since it wouldn't start to precompile stuff that may need to be changed when new constraints come from installing OrdinaryDiffEq.jl.

Reply:
Agreed. Before, we deliberately kept them separate before to make it easier to install only the minimum set of packages. However, I am happy to change that if you feel like it would be more important to reduce precomp time.

Reply:
I think it would be nice to improve the experience. We can of course describe which packages are required for which part and tell people that they can omit packages from the list if they don't intend to use them (in the text).

Reply:
Done in c431e5c0e.